File: ARCADE/Intro/Level 6_5: chessBoardCellColor.cs
    using System;
     
    public class Test
    {
    	public static bool chessBoardCellColor(string cell1, string cell2){
            char column1 = cell1[0], column2 = cell2[0];
            char row1 = cell1[1], row2 = cell2[1];
            if ((column1 + row1) % 2 == (column2 + row2) % 2) return true;
            else return false;
        }
    	public static void Main()
    	{
    		string cell1 = "A1", cell2 = "C3";
    		Console.WriteLine(chessBoardCellColor(cell1,cell2));
    		string cell3 = "A1", cell4 = "H3";
    		Console.WriteLine(chessBoardCellColor(cell3,cell4));
    	}
    }

File: ARCADE/Intro/Level 6_5: chessBoardCellColor.pas
program chessBoardCellColor;
 
function chessBoardCellColor(cell1: string; cell2: string): boolean;
var r1, r2, c1, c2 : char;
    a, b : integer;
begin
    r1 := cell1[1]; r2 := cell2[1];
    c1 := cell1[2]; c2 := cell2[2];
    a := (ord(r1) + ord(c1)) mod 2;
    b := (ord(r2) + ord(c2)) mod 2;
    if(a = b) then chessBoardCellColor := true
    else chessBoardCellColor := false;
end;
 
var cell1, cell2 : string;
begin
	cell1 := 'A1'; cell2 := 'C3';
	writeln(chessBoardCellColor(cell1, cell2));
	cell1 := 'A1'; cell2 := 'H3';
	writeln(chessBoardCellColor(cell1, cell2));
end.
